app/controllers/v1/mods_controller.rb

```diff
@@ -0,0 +1,11 @@
+module V1
+  class ModsController < ApplicationController
+    # Given a POST body with cocina, transform it to MODS xml. Used by Argo to preview metadata
+    def create
+      public_cocina = Cocina::Models.build(params[:mod].to_unsafe_h)
```

Comment:
Is `:mod` a typo?

Reply:
Nope, it's how rails is automatically de-pluralizing "mods"